presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -352,12 +353,6 @@ private PlanRoot doAnalyzeQuery()
         // time analysis phase
         long analysisStart = System.nanoTime();
```

Comment:
Should this be moved as well?

Reply:
I think we need to break up analysis vs planning time. Right now, it mixes both, although planning is the bulk of it.